services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db.models import QuerySet
+from django.utils.datetime_safe import datetime
+
+from db.models import Ticket, Order, User, MovieSession
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(tickets: list[Ticket],
+                 username: str,
+                 date: str = None) -> QuerySet:
+    user = User.objects.get(username=username)
+
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
+        order.save(update_fields=["created_at"])
```

Comment:
This is a case of repeating yourself. You are calling the `save()` method twice: once in line 14 and once in line 17. You can optimize this by setting the `created_at` attribute before creating the order, and then calling the `save()` method only once.